### DISCUSSION TITLE
I can't edit the code or the readme file in my repo

### DISCUSSION THREAD

**Discussion by alimaatouq** [community/community#32822] [Other Feature Feedback, Questions, & Ideas]
Why can't I edit the code or the read me file? The commit button is grayed out please help.

**Comment by augustoaraujoo** [comment]
you must add a name for the commit, or just click on the inputs text

**Reply by augustoaraujoo** [reply]
and you need to modify the file, add some text, for example

**Reply by alimaatouq** [reply]
Hi still not working, I tried on another account and works perfectly, but on my main account it is not. 
Thank you.

**Comment by SharfShiplo** [comment]
When I use the Github web file editor to edit the README.md for my project, the 'Commit message' field is pre-filled with "Update README.md", which is good for me. However, the 'Commit Changes' button is disabled, and I need to modify the field in order to be able to commit the new version of the edited file.

Steps to reproduce

Go to your github project's homepage at https://github.com/username/projectname

Click on the little pencil on the right to edit the README.md file
You're now at https://github.com/username/projectname/edit/master/README.md

Change the text in the file (not in the commit message)
Try clicking the commit button. It's disabled!
Modify the commit message, then the button will be enabled


Expected behavior
The commit message looks pre-filled and its content is OK for my edit. I'd expect the commit changes button to be usable without having to provide a custom commit message...

**Reply by SharfShiplo** [reply]
![01](https://user-images.githubusercontent.com/48642145/189956806-6beb8f04-58c2-4c81-a337-066aa4019824.png)
![02](https://user-images.githubusercontent.com/48642145/189956815-0446bb2f-30b0-4ff4-8245-9586c938347d.png)

**Reply by alimaatouq** [reply]
Hi still not working, I tried on another account and works perfectly, but on my main account it is not. 
Thank you.

**Comment by jfullstackdev** [comment]
I cannot reproduce the error, I went to one of my projects, then hit the Edit button of README, then just typed `x` and the button is now enabled, I'm using Chrome, if you have any extensions, then try Incognito mode, as it is the fresh copy of your browser

![image](https://user-images.githubusercontent.com/47092464/189946201-85c72a7b-be28-4576-8486-7f71cadbab5c.png)

**Reply by alimaatouq** [reply]
Hi still not working, I tried on another account and works perfectly, but on my main account it is not. 
Thank you.
